bootstrap.sh

```diff
@@ -14,6 +14,14 @@ if ! command -v apt-get &>/dev/null; then
   exit 1
 fi
 
+for CMD in curl sudo
+do
+  if ! command -v ${CMD} &>/dev/null; then
+    echo "Error: ${CMD} not found." >&2
+    exit 1
+  fi
+done
+
 apt-get update
 which curl > /dev/null || apt-get install -qq -y curl
```

Comment:
There's already a check for curl so this one isn't needed.

@josegonzalez Best to assume as little as possible, getting an early error message when sudo is missing is better than some command failing.